feature/bgp/gracefulrestart/ate_tests/bgp_graceful_restart_test/bgp_graceful_restart_test.go

```diff
@@ -460,6 +463,217 @@ func configACLInterface(t *testing.T, iFace *oc.Acl_Interface, ifName string) *a
 	return aclConf
 }
 
+// Helper function to replicate configACL() configs in NOKIA native model
```

Comment:
Remove the Nokia keyword.  "in vendor native/CLI model".

The func has a internal switch on vendors. The comment should not just mention Nokia.